src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  src: url(./fonts/Avenir-Book.ttf);
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background-color: #ffff;
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input-small {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.input::placeholder {
+  font-weight: 400;
```

Comment:
remove

Reply:
Without "font-weight" in "::placeholder" text "Try "LA" changes, when focusing on it. It's ok, or how can I fix it? ![image](https://user-images.githubusercontent.com/123917348/225776822-858c796f-78f4-4629-b57d-8d837b6f7f6a.jpeg)![Uploading 225776836...]

Reply:
Without "text-weight:400" in placeholder text "Try LA" changes to bold, when I focus on "input". 🥲 Is it ok or how can I fix it? Please 🙏🏻🙏🏻   ![image](https://user-images.githubusercontent.com/123917348/225838712-e7107ed6-8349-4fbe-9ba5-7884b39abd04.jpeg) ![image](https://user-images.githubusercontent.com/123917348/225838946-60f042df-f86b-4804-a844-200d6678e3ca.jpeg)